runtime/parachains/src/dmp.rs

```diff
@@ -30,6 +34,7 @@ pub use pallet::*;
 mod tests;
 
 pub const MAX_MESSAGE_QUEUE_SIZE: usize = 1024;
+pub const MULTIPLICATIVE_FEE_FACTOR: FixedU128 = FixedU128::from_rational(101, 100); // 1.01
```

Comment:
```suggestion
pub const EXPONENTIAL_FEE_BASE: FixedU128 = FixedU128::from_rational(101, 100); // 1.01
```
? Since it is actually exponential.

Reply:
Yeah sounds reasonable!